website/docs/language/stacks/reference/tfdeploy.mdx

```diff
@@ -41,6 +41,8 @@ Each Stack must have at least one `deployment` block, and the label of the `depl
 | :---- | :---- | :---- | :---- |
 | `inputs` | A mapping of Stack variable names for this deployment.  The keys in this map must correspond to the names of variables defined in the Stack. The values must be valid HCL literals meeting the type constraint of those variables. | map | Required |
 
+### Reference
```

Comment:
All the changes in this file are adding "Reference" headers above examples (so no actual example changes) as per Adam's request!